src/api/app/views/webui/package/_breadcrumb_items.html.haml

```diff
@@ -11,16 +11,16 @@
       = link_to(@package, package_show_path(@project, @package))
     - else
       = link_to truncate(@package.to_s, length: 50), package_show_path(@project, @package)
-  - if current_page?(package_add_file_path)
+  - if current_page?(package_add_file_path(@project, @package))
```

Comment:
This change and the others in this file are needed due to the route parameters being differently name in the context of `role_additions_controller.rb`. `:project` and `:package` are both suffixed by `_name` in this case.